includes/template-library/sources/local.php

```diff
@@ -1390,16 +1328,25 @@ public function add_filter_by_category( $post_type ) {
 	 *
 	 * Import template from a file to the database.
 	 *
-	 * @since 1.6.0
-	 * @access private
-	 *
-	 * @param string $file_name File name.
+	 * @param array $file File name.
 	 *
 	 * @return \WP_Error|int|array Local template array, or template ID, or
 	 *                             `WP_Error`.
+	 *@since 1.6.0
+	 * @access private
+	 *
 	 */
-	private function import_single_template( $file_name ) {
-		$data = json_decode( file_get_contents( $file_name ), true );
+	public function import_single_template( array $file ) {
```

Comment:
What happened here?

Reply:
I added validation, the `json_decode()` call was moved past the validation to line 1348.